src/moveFile.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const fs = require('fs/promises');
+
+async function moveFile(src, dest) {
```

Comment:
If you create function `moveFile` make sense all checking write in this function. Because right now you have logic in two different places and if in the future someone wants to use your function without checking, we will have errors. So, unite that logic for usability.